tests/features/frontend_cache_purge.feature

```diff
@@ -553,7 +553,6 @@ Feature:
     And I press "Save"
     Then the web front end cache was not instructed to purge any paths
     And a critical error message is logged with type "nexteuropa_varnish" and a message matching "No path has been sent for clearing because all module settings are not set."
-    And no informational message is logged with type "nexteuropa_varnish" and a message matching "Clearing paths: /more-basic-pages, /, /even-more-basic-pages"
```

Comment:
I do not understand this conflict fix. I think that we introduce a regression bug with this PR.
We need to resolve the situation, and for me, it's not good to have these two messages in the same time in the logs.